eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/boot/EventMeshAdminBootstrap.java

```diff
@@ -17,34 +17,28 @@
 
 package org.apache.eventmesh.runtime.boot;
 
-import org.apache.eventmesh.runtime.admin.controller.HttpHandlerManager;
+import org.apache.eventmesh.runtime.admin.controller.ClientManageController;
 import org.apache.eventmesh.runtime.configuration.EventMeshHTTPConfiguration;
```

Comment:
I feel that referencing `ClientManageController` in `EventMeshAdminBootstrap` is not a good practice. I think the design of `ClientManageController` and `HttpHandlerManager` has been outdated and it will be great if you may redesign them to fit EventMesh style of Netty HTTP Server.

Reply:
That's right,please review it.